kuma/static/js/utils/perf.js

```diff
@@ -47,10 +47,15 @@ mdn.perf = {
             return;
         }
 
-        performance.measure(
-            measureData.measureName,
-            measureData.startMark,
-            measureData.endMark
-        );
+        try {
+            performance.measure(
+                measureData.measureName,
+                measureData.startMark,
+                measureData.endMark
+            );
+        } catch (error) {
+            console.error('Error while setting performance measure: ', error);
```

Comment:
```suggestion
            console.error(`Error while setting performance measure: ${error.toString()}`);
```
Your code is fine but I wanna take this opportunity to better understand Interactive Examples and to carefully see if this change fixes things.

Reply:
Also, I would love to use JS template strings, but our pipeline does not allow us at the moment. If you run `build-static` you will see it throws an error:

```
pipeline.exceptions.CompressorError: b"Parse error at -:57,26\n            console.error(`Error while setting performance measure: ${error.toSt\n                          ^\nSyntaxError: Unexpected character '`'\n    at JS_Parse_Error.get (eval at <anonymous> (/tools/node_modules/uglify-js/tools/node.js:27:1), <anonymous>:86:23)\n    at /tools/node_modules/uglify-js/bin/uglifyjs:384:40\n    at time_it (/tools/node_modules/uglify-js/bin/uglifyjs:620:15)\n    at /tools/node_modules/uglify-js/bin/uglifyjs:345:9\n    at Socket.<anonymous> (/tools/node_modules/uglify-js/bin/uglifyjs:601:13)\n    at Socket.emit (events.js:215:7)\n    at endReadableNT (_stream_readable.js:1184:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)\n"
```